packs/abomination-vaults-bestiary/book-2-hands-of-the-devil/barcumbuk.json

```diff
@@ -1415,7 +1415,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>An erinys carries a coil of rope woven of their hair (use the statistics for @UUID[Compendium.pf2e.equipment-srd.Item.Rope]) that animates in their hands.</p>\n<p>When a creature is hit by the erinys's rope, a segment of the rope tears loose and wraps itself around the creature, imposing a 10-foot circumstance penalty to Speed. The piece that tears off is 10 feet long for a Medium or smaller creature, and doubles in length for each size larger than Medium.</p>\n<p>When a creature @UUID[Compendium.pf2e.actionspf2e.Item.Escape]{Escapes} the effect (DC 26), the detached segment of rope withers away into useless black sludge.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Rope Snare]</p>"
+                    "value": "<p>An erinys carries a coil of rope woven of their hair (use the statistics for @UUID[Compendium.pf2e.equipment-srd.Item.Rope]) that animates in their hands.</p>\n<p>When a creature is hit by the erinys's rope, a segment of the rope tears loose and wraps itself around the creature, imposing a 10-foot circumstance penalty to Speed. The piece that tears off is 10 feet long for a Medium or smaller creature, and doubles in length for each size larger than Medium.</p>\n<p>When a creature @UUID[Compendium.pf2e.actionspf2e.Item.Escape]{Escapes} the effect ([[/act escape dc=26]]{DC 26}), the detached segment of rope withers away into useless black sludge.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Rope Snare]</p>"
```

Comment:
Changes to escape actions should probably wait until after the issue of showing the DC inline is resolved?